### ISSUE TITLE
Demo shows random predictions.

### ISSUE THREAD

**Issue by Samb55** [matterport/Mask_RCNN#1353]
Hello everyone.
I got 22% training loss and 23% validation loss. But, when I ran the demo on my test set I got random predictions. No one of these predictions was a good result. However, when I ran inspect_model I got good predictions. Please anyone can tell me why? How I improve it?
